### PR TITLE
Remove SB parent (redhat)

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -18,19 +18,63 @@
...
+      <dependency>
+        <groupId>org.jboss.arquillian</groupId>
+        <artifactId>arquillian-bom</artifactId>
+        <version>${arquillian.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>junit</groupId>
+        <artifactId>junit</artifactId>
```
Comment (gytis) [inline pom.xml]: junit doesn't seem to be used directly. Other than that all good.

Comment (geoand) [reply pom.xml]: I'll keep it for the moment. We can cleanup in a later PR. Does that sound ok?

Comment (gytis) [reply pom.xml]: OK. The same for the other boosters then.

Comment (geoand) [reply pom.xml]: Cool

**Pull request merged**